test/e2e/kibishii_tests.go

```diff
@@ -104,23 +119,23 @@ func RunKibishiiTests(client *kubernetes.Clientset, providerName, veleroCLI, vel
 		return errors.Wrap(err, "Failed to generate data")
 	}
 
-	if err := VeleroBackupNamespace(oneHourTimeout, veleroCLI, veleroNamespace, backupName, kibishiiNamespace, backupLocation, useVolumeSnapshots); err != nil {
-		VeleroBackupLogs(fiveMinTimeout, veleroCLI, veleroNamespace, backupName)
+	if err := veleroBackupNamespace(oneHourTimeout, veleroCLI, veleroNamespace, backupName, kibishiiNamespace, backupLocation, useVolumeSnapshots); err != nil {
```

Comment:
These were all public so we didn't need to put all of the e2e tests in the same package.  Eventually I think we want to split this up.

Reply:
When we do split these tests into separate packages will be a great time to export what is needed to be exported.